src/main/java/com/gmail/inverseconduit/bot/InteractionBot.java

```diff
@@ -26,13 +26,12 @@ public InteractionBot(ChatInterface chatInterface) {
 
     @Override
     public void start() {
-        executor.scheduleAtFixedRate(this::processInteractions, 200, 700, TimeUnit.MILLISECONDS);
+        // asynchronously enqueue the processing by blocking supplier
+        processingThread.submit(this::processInteractions);
```

Comment:
So you'd rather have it wait indefinitely? I'll go look through the Queues if I can find something that blocks non-periodically and use that instead, then